File: Programmers/Level1/크레인인형뽑기게임/크레인인형뽑기게임.py
def solution(board, moves):
    result = []
    answer = 0
    for m in moves:
        for i in range(len(board)):
            if board[i][m - 1] != 0:
                result.append(board[i][m - 1])
                board[i][m - 1] = 0
                
                if len(result) > 1:
                    if result[-1] == result[-2]:
                        answer += 2
                        result = result[:-2]
                break
    return answer

File: Programmers/Level1/크레인인형뽑기게임/크레인인형뽑기게임.js
function solution(board, moves) {
    let result = [],
        answer = 0;

    for (let m = 0; m < moves.length; m++) {
        for (let i = 0; i < board.length; i++) {
            if (board[i][moves[m] - 1] !== 0) {
                result.push(board[i][moves[m] - 1]);
                board[i][moves[m] - 1] = 0;

                if (
                    result.length != 1 &&
                    result[result.length - 1] === result[result.length - 2]
                ) {
                    answer += 2;
                    result = result.slice(0, -2);
                }
                break;
            }
        }
    }

    return answer;
}
